src/components/atoms/title/title.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { render, screen } from '@testing-library/react';
+import { Title } from './title';
+
+describe('Title', () => {
+  it('should render the title with a string passed into the title prop', () => {
+    const title = 'This is a title';
+    render(<Title title={title}/>);
+
+    expect(screen.getByText(title)).toBeInTheDocument();
+  });
+
+  it('should render the title with a array of strings passed into the title prop', () => { // whitespace issues
+    const titles = ['This', 'is', 'a', 'title'];
+    render(<Title title={titles}/>);
+
+    expect(screen.getByText('Thisisatitle')).toBeInTheDocument();
+  });
+
+  it('should render the title with a single content part passed into the title prop', () => {
+    const title = { type: 'Strong', content: 'This is a title' };
+    render(<Title title={title}/>);
+
+    expect(screen.getByText('This is a title')).toBeInTheDocument();
+  });
+
+  it('should render the title with an array of content parts passed into the title prop', () => { // whitespace issues
+    const title = [{ type: 'Strong', content: 'This is a' }, { type: 'Emphasis', content: 'title' }];
+    render(<Title title={title}/>);
+
+    expect(screen.getByText('This is a')).toBeInTheDocument();
+    expect(screen.getByText('title')).toBeInTheDocument();
+  });
+
+  it('should render the title with an array of content parts and strings passed into the title prop', () => { // components in a list need a key
```

Comment:
@will-byrne Not sure I understand what this means? Is it a React limitation?

Reply:
yes anything in react that is an output of a loop needs a key so react can link the dom / shadow dome elements